website/blog/2017-09-26-adding-rss.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Do we have to remove the demo blog posts?

Reply:
Do you want me to remove this, or...? I'm not really sure.

Reply:
I don't know docusaurus well enough - do you think we should keep them?